docs/governance/overview.md

```diff
@@ -2,29 +2,31 @@
 
 ---
 
-Decred’s governance is based on the principle of ticket-holder voting. To participate in governance, users must time-lock DCR to purchase [tickets](../proof-of-stake/overview.md). Tickets can then be used to vote on validating blocks, consensus rule changes, or proposals in Decred’s web-based proposal system, [Politeia](https://proposals.decred.org/). 
+Decred is a Decentralized Autonomous Organization (DAO). Changes to the system, for example consensus rule changes, happen in a decentralized manner. Decred’s governance is based on the principle of ticket-holder voting. To participate in governance, users must time-lock DCR to purchase tickets. There are multiple things ticket holders can vote on. Some of them are on-chain, and some of them are off-chain.
 
+## On-Chain Voting
 Validation of blocks and voting on consensus rule changes occurs on-chain. Voting on higher-level issues, such as how to spend [Treasury funds](https://dcrdata.decred.org/address/Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx), or significant policy decisions (e.g. amend the [Decred constitution](decred-constitution.md)) occurs off-chain in Politeia, a web platform where the community at large can submit, discuss and vote on proposals.
 
-To compensate ticket holders for validating blocks and securing the network, 30% of the block reward (PoS reward) is sent to ticket holders for voting on blocks; the remaining block reward is split between PoW miners (60%) and the Treasury (10%) to fund ongoing development. The ticket price (the amount of DCR that must be time-locked) is adjusted dynamically every 144 blocks (~12 hrs), in an attempt to keep PoS returns stable over time. 
+On-chain voting serves two main purposes: validating blocks created by Proof-of-Work (PoW) miners and voting on proposed consensus rule changes.
 
+### Block Voting
 
----
-
-## On-chain voting
+In each block, five tickets are selected pseudo-randomly and called to [vote on-chain](../proof-of-stake/overview.md). Tickets vote to approve or reject the previous block of transactions created by a PoW miner. At least three out of five tickets called to vote must respond for the block to be considered valid. This power provides a check against malicious PoW miner behavior (e.g. mining empty blocks), as well as forming the basis of Decred’s [fork resistance](https://medium.com/decred/detailed-analysis-of-decred-fork-resistance-93022e0bcde7). After a ticket is called to vote, the DCR locked in that ticket is unlocked and returned to the purchaser’s wallet, along with the PoS reward if the ticket successfully voted. Decred is a hybrid PoW/PoS system.
```

Comment:
So this is slightly wrong.  Each block MUST have at least 3 votes to be mined at all.

And here is a list of the various voting situations and the resulting previous block validation: http://zer0byte.com/zeropastebin/?a15be573eb4c32f4#D/2fHCHIpjYiL1iIEw8Tu5dsLwkLNM8HjyNc9c64YKc=